src/layouts.js

```diff
@@ -57,6 +57,11 @@ function setTheme(context, removeTheme = false){
 
     // set theme
     context.outer.setAttribute(ATTRS.theme, context.options.theme);
+
+    if(context.isCornerTheme){
+        const formatPosition = context.options.position.trim().split(" ").join("-");
```

Comment:
Create `CORNER_POSITIONS` array in the config that contains the allowed value of `position`. Throw a warning and return it to the default option if the unallowed option is entered.